src/main/webapp/js/grapheditor/Graph.js

```diff
@@ -28,8 +28,10 @@ function parseStyle(style) {
   const shape = jsonStyle.shape;
   if (shape) {
     const customProperties = mxCellRenderer.defaultShapes[shape].prototype.customProperties;
-    for (const {name, defVal} of customProperties) {
-      jsonStyle[name] = jsonStyle[name] || defVal;
+    if (customProperties != null) {
+      for (const {name, defVal} of customProperties) {
+        jsonStyle[name] = jsonStyle[name] || defVal;
```

Comment:
I think that way we cant set a property to `false` if its `defVal` is `true, maybe something like:
```suggestion
        if (typeof(jsonStyle[name]) === 'undefined') { 
          jsonStyle[name] = defVal;
        }
```

Reply:
Fixed :)

Reply:
sorry, but don't think so. We still have the problem of setting a property to `true` if its `defVal` is `true` while we are setting it to `false`.

Reply:
I changed the wrong part of the code :P I'll wait for your thumbs up before setting as resolved this time.